container/state.go

```diff
@@ -278,6 +278,7 @@ func (s *State) SetRunning(pid int, initial bool) {
 	s.ErrorMsg = ""
 	s.Running = true
 	s.Restarting = false
+	s.Paused = false
```

Comment:
Would there be a situation where the container is actually paused, and this function is called? Should calling this function be an error? i.e.;

```bash
$ docker pause 07e9ed84b928
07e9ed84b928

$ docker start 07e9ed84b928
Error response from daemon: Cannot start a paused container, try unpause instead.
Error: failed to start containers: 07e9ed84b928
```